x/circuit/keeper/msg_server.go

```diff
@@ -37,6 +37,10 @@ func (srv msgServer) AuthorizeCircuitBreaker(ctx context.Context, msg *types.Msg
 		// Check that the authorizer has the permission level of "super admin"
 		perms, err := srv.Permissions.Get(ctx, address)
 		if err != nil {
+			if errorsmod.IsOf(err, collections.ErrNotFound) {
```

Comment:
The error is quite ugly otherwise.

Reply:
i thought this was fixed? something was removed and this was removed from other places

Reply:
No, you are thinking of another fix on the walk iterator (https://github.com/cosmos/cosmos-sdk/issues/17289).
This quite makes sense to return not found. We have such checks at other places in the SDK (e.g. https://github.com/cosmos/cosmos-sdk/blob/45b44e678ccb9dcf6c1284a36632618da94e9567/x/staking/keeper/unbonding.go#L68), to return a better human-readable error instead.